clients/drdisas/drdisas.cpp

```diff
@@ -125,22 +143,55 @@ main(int argc, const char *argv[])
 
     void *dcontext = GLOBAL_DCONTEXT;
 
-#if defined(X86_64) || defined(ARM)
+    dr_disasm_flags_t syntax =
+#ifdef X86
+        DR_DISASM_INTEL
+#elif defined(RISCV64)
+        DR_DISASM_RISCV
+#else
+        DR_DISASM_DR
+#endif
+        ;
+
     // Set the ISA mode if supplied.
```

Comment:
Here something like: "Set the ISA mode if supplied. Overwrite the default syntax to DR_DISASM_DR when the user-provided ISA mode is the synthetic 'regdeps' ISA."

Reply:
Done.